### ISSUE TITLE
Transaction from Spark to transparent is not getting confirmed

### ISSUE THREAD

**Issue by nrsimha** [firoorg/firo#1460]
### Describe the issue

Sent 1000 Firo from Spark to transparent address and it is completely ignored by mined blocks (almost 1 hour), doesn't get confirmed.
Tried to re-broadcast, but still getting ignored. Status: 0/unconfirmed, in memory pool
Before re-broadcast it did show status mentioning broadcasted to 8 nodes or something like this.

### Local setup

Macos Ventura
Firo Core version v0.14.13.3-gd3722c66e (64-bit)

**Comment by nrsimha** [comment]
Seems issue transaction is too big.
In latest version transaction size was increased, so it doesn't show error, but older version which is still running on Allnodes and Pecunia has limit 256kB.

**Comment by justanwar** [comment]
```
$ firo-cli getpeerinfo | grep "0.14.13.2" -c
28
$ firo-cli getpeerinfo | grep "0.14.13.3" -c
10
```
Looks like there's still not enough peers updated to v0.14.13.3.

**Comment by justanwar** [comment]
Should be fixed by now as there's a lot more v0.14.13.3 peers.
